spec/refresh_pools_spec.rb

```diff
@@ -43,6 +43,15 @@
     end
   end
 
+  it 'contains the proper return value when owner is auto generated' do
+    result = @cp.refresh_pools("auto-generated-owner", false, true)
+    if !is_hosted?
+      expect(result).to be_nil
+    else
+      expect(result).to eq("Pools refreshed for owner: auto-generated-owner")
```

Comment:
let's also fetch the owner object itself to verify it's there, and assert that it's contentAccessMode is set to org_environment

Reply:
Done